modules/planning/common/BUILD

```diff
@@ -18,13 +18,23 @@ cc_library(
 cc_library(
     name = "logic_map",
     srcs = [
+        "logic_node.cc",
         "logic_point.cc",
     ],
     hdrs = [
+        "logic_node.h",
         "logic_point.h",
     ],
     deps = [
         "//modules/common/math",
+        "//modules/planning/common:planning_error_code",
```

Comment:
let's use the error code in the apollo/common. Please move the error code declaration together with other error code management.